src/common/box_filters.c

```diff
@@ -1272,14 +1272,22 @@ void dt_box_mean_vertical(float *const buf, const size_t height, const size_t wi
     dt_unreachable_codepath();
 }
 
+static inline float window_max(const float *x, int n)
+{
+  float m = -(FLT_MAX);
+#ifdef _OPENMP
+#pragma omp simd reduction(max : m)
+#endif
+  for(int j = 0; j < n; j++)
+    m = MAX(m, x[j]);
```

Comment:
why not `fmaxf` ?

Reply:
I've found numerous times that fminf/fmaxf don't vectorize while MIN/MAX do.

Reply:
ok

Reply:
ok